parser_test.go

```diff
@@ -404,7 +404,8 @@ const exampleStateFile = `
 							"access_ip_v4": "10.0.0.19",
 							"access_ip_v6": "",
 							"tag.%": "1",
-							"tag.tfinventory": "rocks"
+							"tag.tfinventory": "rocks",
+                            "metadata.superman": "clarkkent"
```

Comment:
Please fix indentation. This file uses tabs.

Reply:
Good catch, thanks! It's fixed now. :)